tcms/testcases/tests/test_views.py

```diff
@@ -59,9 +59,10 @@ def test_user_in_default_group_sees_comments(self):
             for status in TestExecutionStatus.objects.all():
                 self.assertContains(
                     response,
-                    "<input type=\"submit\" class=\"btn btn_%s btn_status js-status-button\" "
-                    "title=\"%s\"" % (status.name.lower(), status.name),
-                    html=False
+                    '<i class="submit_button %s fa-2x" style="color: %s" \
+                        statusId="%s" title="%s"></i>'
+                    % (status.icon, status.color, status.pk, status.name),
+                    html=True
```

Comment:
Status names are translated and this is why the test probably fails. Remove the `override` statement above this block and it should pass.